models/comments-models.js

```diff
@@ -0,0 +1,14 @@
+const db = require("../db/connection");
+
+exports.checkArticleExists = (article_id) => {
```

Comment:
I could be mistaken but don't you already have a model that fetches an article by id that you could use?